projects/core/src/util/external-js-file-loader/external-js-file-loader.service.ts

```diff
@@ -20,6 +23,9 @@ export class ExternalJsFileLoader {
     callback?: EventListener,
     errorCallback?: EventListener
   ): void {
+    if (isPlatformServer(this.platformId)) {
+      return;
```

Comment:
I'm wondering about sth like:
```
    if (errorCallback) {
      errorCallback(new Event('error'));
    }
```
before return, any opinions?

Reply:
good idea, will add it.